package.json

```diff
@@ -54,7 +54,7 @@
     "watch": "gulp watch",
     "coverage": "gulp coveralls",
     "precommit-msg": "echo 'Pre-commit checks...' && exit 0",
-    "precommit-test": "REPORTER=dot gulp once",
+    "precommit-test": "env REPORTER=dot bash -c 'gulp once'",
```

Comment:
why this change is necessary?

Reply:
on my PC (Windows10) I have permanent error when do commit. precommit hook:
appium-windows-driver> git.exe commit --file=C:/Users/autotest/AppData/Local/Temp/smartgit-675290320427002764tmp/commit-12634442983062762856.tmp -o -- package.json
'Pre-commit checks...' 
'REPORTER' is not recognized as an internal or external command,
operable program or batch file.
pre-commit: 
pre-commit: We've failed to pass the specified git pre-commit hooks as the `precommit-test`
pre-commit: hook returned an exit code (1). If you're feeling adventurous you can
pre-commit: skip the git pre-commit hooks by adding the following flags to your commit:
pre-commit: 
pre-commit:   git commit -n (or --no-verify)
pre-commit: 
pre-commit: This is ill-advised since the commit is broken.
pre-commit:

Reply:
@imurchie FYI